pillar/heroku/bootcamps.sls

```diff
@@ -65,41 +67,42 @@
       'FEATURE_NOVOED_INTEGRATION': True,
       'GA_TRACKING_ID': 'UA-5145472-18',
       'GTM_TRACKING_ID': 'GTM-TFSZHVB',
+      'HUBSPOT_API_ID_PREFIX' 'bootcamp',
       'HUBSPOT_PORTAL_ID': '6119748',
       'HUBSPOT_FOOTER_FORM_GUID': '2d798908-c195-4c0c-b075-a10b0c1b08f3',
+      'JOBMA_BASE_URL': 'https://www.jobma.com',
       'MAILGUN_SENDER_DOMAIN': 'mail.bootcamp.odl.mit.edu',
       'NOVOED_BASE_URL': 'https://mitbootcamps.novoed.com',
       'NOVOED_SAML_DEBUG': False,
       'NOVOED_SAML_LOGIN_URL': 'https://app.novoed.com/saml/sso?provider=mitbootcamps',
       'SITE_NAME': 'MIT Bootcamps',
-      'vault_env_path': 'production-apps'
       }
 } %}
 {% set env_data = env_dict[environment] %}
-{% set business_unit = 'bootcamps' %}
-{% set cybersource_creds = salt.vault.read('secret-' ~ business_unit ~ '/' ~ env_data.vault_env_path ~ '/cybersource').data %}
-{% set jobma = salt.vault.read('secret-' ~ business_unit ~ '/' ~ env_data.vault_env_path ~ '/jobma').data %}
+{% set cybersource_creds = salt.vault.read('secret-bootcamps/data/cybersource').data %}
+{% set jobma = salt.vault.read('secret-bootcamps/data/jobma').data %}
+{% set mit_smtp = salt.vault.read('secret-global/data/mit-smtp').data %}
 {% set rds_endpoint = salt.boto_rds.get_endpoint('bootcamps-db-applications-{env}'.format(env=env_data.aws_env)) %}
 
 proxy:
   proxytype: heroku
 
 heroku:
   app_name: {{ env_data.app_name }}
-  api_key: __vault__::secret-operations/global/heroku/mitx-devops-api-key>data>value
+  api_key: __vault__::secret-global/data/heroku>data>data>mitx-devops-api-key
   config_vars:
     ALLOWED_HOSTS: '{{ env_data.ALLOWED_HOSTS|tojson }}'
-    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/bootcamps-app>data>access_key
-    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/bootcamps-app>data>secret_key
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/bootcamps-app>data>>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/bootcamps-app>data>data>secret_key
     AWS_STORAGE_BUCKET_NAME: 'ol-bootcamps-app-{{ env_data.aws_env }}'
     BOOTCAMP_ADMIN_EMAIL: cuddle-bunnies@mit.edu
     BOOTCAMP_DB_DISABLE_SSL: True
     BOOTCAMP_ECOMMERCE_BASE_URL: {{ env_data.BOOTCAMP_ECOMMERCE_BASE_URL }}
-    BOOTCAMP_EMAIL_HOST: __vault__::secret-operations/global/mit-smtp>data>relay_host
-    BOOTCAMP_EMAIL_PASSWORD: __vault__::secret-operations/global/mit-smtp>data>relay_password
+    BOOTCAMP_EMAIL_HOST: {{ mit_smtp.relay_host }}
+    BOOTCAMP_EMAIL_PASSWORD: {{ mit_smtp.relay_password }}
```

Comment:
This will likely require another layer of `.data` attribute access if it's a V2 mount.